spread/lxd.go

```diff
@@ -67,6 +95,44 @@ func (s *lxdServer) Discard(ctx context.Context) error {
 	return nil
 }
 
+func (s *lxdServer) Prepare(ctx context.Context) error {
+	if s.Distro() == Unknown {
+		s.Discard(ctx)
+		return fmt.Errorf("Unknown distro type for image %s, bailing", s.System().Image)
+	}
+
+	args := []string{"exec", s.d.Name, "--"}
+	args = append(args, sshInstallCommand(s.Distro())...)
+
+
+	output, err := exec.Command("lxc", args...).CombinedOutput()
+	if err != nil {
+		printf("Command output: %s", output)
+		s.Discard(ctx)
+		return err
+	}
+
+	err = s.p.tuneSSH(s.d.Name, s.Distro())
+	if err != nil {
+		s.Discard(ctx)
+		return err
+	}
+
+	if s.Distro() == Fedora {
+		// Fedora LXD images do *not* contain tar by default, so we must install it manually
+		args = []string{"exec", s.d.Name, "--", "dnf", "install", "--assumeyes", "tar"}
```

Comment:
As discussed, this could be done by adding tar to the [fedora image setup script](https://github.com/lxc/lxc-templates/blob/master/templates/lxc-fedora.in#L610) instead.